save-backend/src/main/kotlin/com/saveourtool/save/backend/configs/ConfigProperties.kt

```diff
@@ -15,6 +16,7 @@ import org.springframework.boot.context.properties.ConstructorBinding
 @ConstructorBinding
 @ConfigurationProperties(prefix = "backend")
 data class ConfigProperties(
+    val url: String,
```

Comment:
is it a new property, why is it needed here?

Reply:
yes, it's a new property, to provide an url to backend. it's used in `InitConfig` in the first heartbeat response

Reply:
why we should send heartbeat to backend?

Reply:
we don't send hearbeat to backend, backend sends `InitConfig` to orchestrator with information to run execution.
It contains links to download tests, additional files and save-cli
It needs for hot-reloading and migration to S3